pkg/logging/fields.go

```diff
@@ -9,20 +28,27 @@ func Err(err error) zap.Field {
 
 // ImageName provides the image name as a structured log field.
 func ImageName(name string) zap.Field {
-	return zap.String("image", name)
+	return zap.String(imageField, name)
 }
 
 // ClusterID provides the cluster ID as a structured log field.
 func ClusterID(id string) zap.Field {
-	return zap.String("cluster_id", id)
+	return zap.String(clusterIDField, id)
 }
 
 // ImageID provides the image ID as a structured log field.
 func ImageID(id string) zap.Field {
-	return zap.String("image_id", id)
+	return zap.String(imageIDField, id)
 }
 
 // NodeID provides the node ID as a structured log field.
 func NodeID(id string) zap.Field {
-	return zap.String("node_id", id)
+	return zap.String(nodeIDField, id)
+}
+
+// isResourceTypeField returns whether the given zap.Field is related to a resource.
+// If it is, it will return true and the name of the resource.
+func isResourceTypeField(field zap.Field) (bool, string) {
```

Comment:
```suggestion
func getResourceTypeField(field zap.Field) (string, bool) {
```
Seems more golang idomatic to me.